fix #10357

johnrackles opened the pull request:
This changes the text from
 "This widget is only displayed to users having Super User access."
to
"This widget is displayed in the default dashboard only to users having Super User access."

fix #10357

tsteur commented on plugins/CoreHome/lang/de.json:
```diff
@@ -29,7 +29,7 @@
         "Menu": "Menü",
         "NoPrivilegesAskPiwikAdmin": "Sie sind angemeldet als '%1$s', aber es wurden keine Berechtigungen für diesen Benutzer in Piwik gesetzt. %2$s Bitte fragen Sie Ihren Piwik Administrator (klicken für E-Mail)%3$s um Zugriff auf die Webseite zu erhalten.",
-        "OnlyForSuperUserAccess": "Dieses Widget wird nur Benutzern mit Hauptadministrator-Berechtigung angezeigt.",
+        "OnlyForSuperUserAccess": "Dieses Widget wird im standart Dashboard nur Benutzern mit Hauptadministrator-Berechtigung angezeigt.",
```
I think the "Duden" says "standard", maybe use "Standard-Dashboard" as we have used this wording in other translations so far

tsteur commented:
Thank you for creating the pull request 👍 

I should have mentioned in the issue that translations are actually managed via https://www.transifex.com/piwik/piwik/ 

Do you mind having a look there? If there are any problems @sgiehl may be able to help you.

I will close this issue as translations have to be changed via the translation tool. Looking forward to seeing that change in Piwik, cheers.

mattab commented:
To clarify: non english translations are managed on transifex (eg. German strings) but english strings are managed in the Git repository in `en.json` files. @johnrackles could you create this PR only with the `en.json` file change? then you will be able to translate it on transifex into german. Cheers

tsteur commented:
that's good to know. Is it not possible to change the english translation in transifex and it clears all the other translations?

sgiehl commented:
Transifex automatically pulls the translation files from piwik/master (or the plugin repos). If a value changes, all other languages will be cleared automatically for this key.

Commit:
fix #10361

Now only the english string is changed, as translations are handled via https://www.transifex.com/piwik/piwik/

johnrackles commented:
Wow, I do not know what I'm doing with git. But I think I finally managed to update this PR.

tsteur commented:
That looks good 👍  Thx 

@sgiehl does this also happen when changing a value for an English translation in transifex? I know it's working that way when making a change in an English translation file, but thought it would maybe as well work when changing the translation in Transifex?

tsteur commented:
@johnrackles Congrats on closing this issue 👍
